src/ompl/geometric/planners/bitstar/datastructures/src/IntegratedQueue.cpp

```diff
@@ -1586,23 +1586,21 @@ namespace ompl
         bool BITstar::IntegratedQueue::isCostWorseThan(const ompl::base::Cost& a, const ompl::base::Cost& b) const
         {
             //If b is better than a, then a is worse than b
-            return this->isCostBetterThan(b, a);
+            return opt_->isCostBetterThan(b, a);
         }
 
 
 
         bool BITstar::IntegratedQueue::isCostEquivalentTo(const ompl::base::Cost& a, const ompl::base::Cost& b) const
         {
-            //If a is not better than b, and b is not better than a, then they are equal
-            return !this->isCostBetterThan(a,b) && !this->isCostBetterThan(b,a);
+            return (a.value() == b.value());
```

Comment:
It also means that, if you are using the OptimizationObjective::isCostBetterThan function with it's margin in the comparator there is consistency between isCostEquivalentTo and isCostBetterThan. If not you can run into the case where,
`isCostBetterThan(a, b) = false`, and `isCostBetterThan(b,a) = false`, but `isCostEquivalentTo(a,b) = false`. Granted, since I'm not using the comparator with a margin, this isn't a problem, but I'm defensive. ;)